composer.json

```diff
@@ -25,11 +25,11 @@
         "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "^3",
+        "matthiasnoback/symfony-dependency-injection-test": "^4",
```

Comment:
Version 3 is not compatible with PHPUnit 8 due to the infamous `setup(): void` return type addition: https://github.com/SymfonyTest/SymfonyDependencyInjectionTest/pull/109